Dockerfile

```diff
@@ -14,10 +14,17 @@ EXPOSE 8096
 VOLUME /config /media
 
 ARG FFMPEG_URL=https://www.johnvansickle.com/ffmpeg/old-releases/ffmpeg-4.0.3-64bit-static.tar.xz
-RUN apt update \
- && apt install -y xz-utils \
- && curl ${FFMPEG_URL} | tar Jxf - -C /usr/bin --wildcards --strip-components=1 ffmpeg*/ffmpeg ffmpeg*/ffprobe \
- && apt remove -y xz-utils \
- && apt install -y libfontconfig1  # needed for Skia
+RUN DEBIAN_FRONTEND=noninteractive \
```

Comment:
What's the point of this?

Reply:
In this particular instance, nothing. Some packages however pop up with that TUI screen asking you to do stuff. Sometimes even as part of installing a package that previously didn't.

What that does is suppress the TUIs.

Reply:
If it's not needed, please remove it.

Reply:
It's recommended to keep it during automated installations like in Dockerfile, it's not required, but recommended to keep it. The same applies to change replacing `apt` with `apt-get`. In Debian `apt` is used for manual installations where user is present and can control terminal, `apt-get` is used in automated processes.